src/string_util.cc

```diff
@@ -92,21 +91,24 @@ std::string ExponentToPrefix(int64_t exponent, bool iec) {
   const int64_t index = (exponent > 0 ? exponent - 1 : -exponent - 1);
   if (index >= kUnitsSize) return "";
 
-  const char* array =
+  const std::string* array =
       (exponent > 0 ? (iec ? kBigIECUnits : kBigSIUnits) : kSmallSIUnits);
   if (iec) {
-    return array[index] + std::string("i");
+    return array[index] + "i";
   }
-  return std::string(1, array[index]);
+  return array[index];
 }
 
 std::string ToBinaryStringFullySpecified(double value, double threshold,
-                                         int precision, double one_k = 1024.0) {
+                                         int precision, benchmark::Counter::OneK one_k = benchmark::Counter::kIs1024) {
```

Comment:
you should be in the `benchmark` namespace so the `benchmark::` prefix is unnecessary.

Reply:
I corrected this.